.changeset/sharp-boxes-fix.md

```diff
@@ -0,0 +1,22 @@
+---
+"@hopper-ui/tokens": patch
```

Comment:
you need to add 
```
"@hopper-ui/styled-system": patch
```

The tokens also updates the styled-system, as the value of <Div backgroundColor="upsell" will also change